.travis.yml

```diff
@@ -42,7 +42,6 @@ before_script:
   - cat /tmp/supervisord.log
 script:
   - source activate qp-target-gene
-  - start_target_gene "https://localhost:8383" register ignored
```

Comment:
Wouldn't this prevent the plugin from starting?

Reply:
qiita automatically does that, in fact the error that was making the builds fail is that qiita was complaining that the commands already existed ...